Core/src/org/sleuthkit/autopsy/geolocation/datamodel/GeoLocationParseResult.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2020 Basis Technology Corp.
+ * contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.geolocation.datamodel;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.collections4.ListUtils;
+import org.sleuthkit.datamodel.BlackboardArtifact;
+
+/**
+ * The result of attempting to parse a GeoLocation object.
+ */
+public class GeoLocationParseResult<T> {
```

Comment:
Since these are aggregated everywhere and then partition and managed in code outside of this class, what if the granularity of this class was not on a single ParseResult, but multiple? I think that might simplify things, what do you think?

Reply:
Sure.